README.md

```diff
@@ -7,9 +7,20 @@
 
 Source-to-image (`s2i`) is a tool for building reproducible Docker images. `s2i` produces
 ready-to-run images by injecting source code into a Docker image and *assembling*
-a new Docker image which incorporates the builder image and built source.  The result is then ready to use
-with `docker run`. `s2i` supports incremental builds which re-use previously downloaded
-dependencies, previously built artifacts, etc.
+a new Docker image for the application.  The result is then ready to use with `docker run`. 
+
+The basic S2I flow is:
+
+1. Start with a ruby builder image which contains ruby, bundler, rake, apache, gcc, etc, all of which are needed to install and run ruby code.  
+1. S2I launches a container using the builder image and injects the provided source code into a working directory within the container.
+1. The container assembles the source code into a runnable structure.
+1. S2I commits the new container, setting the default run command for the new image to a script that will launch the assembled application.
```

Comment:
Commit the new container and set the image entrypoint to be a script (provided by the builder image) that will start Apache to host the Ruby app.